src/components/button/button.test.js

```diff
@@ -46,9 +46,11 @@ context("Test for Button component", () => {
       "should render Button subtext with %s special characters",
       (subtext) => {
         CypressMountWithProviders(
-          <Button size="large" subtext={subtext}>
-            Foo
-          </Button>
+          <>
+            <Button size="large" subtext={subtext}>
+              Foo
+            </Button>
+          </>
```

Comment:
```suggestion
            <Button size="large" subtext={subtext}>
              Foo
            </Button>
```
This should work without the fragment